pqcrypto-classicmceliece/build.rs

```diff
@@ -136,7 +138,9 @@ fn main() {
                 .flag("-mbmi2")
                 .flag("-mbmi")
                 .flag("-maes")
-                .flag("-mpopcnt");
+                .flag("-mpopcnt")
+                .flag("-mpclmul")
+                .flag("-msse2");
```

Comment:
Why are these flags necessary by the way? 

Ideally they should be read from `META.yml` from PQClean, but this hack will suffice for now.

Reply:
It seems that HQC uses the polynomial carry-less multiplication in order to speed up the Galois Field arithmetic.
The SSE2 flag was suggested by the compiler, but is not necessary. I removed it therefore.